executorWdl/src/main/scala/dx/executor/wdl/WdlTaskExecutor.scala

```diff
@@ -64,9 +64,9 @@ case class WdlTaskExecutor(task: TAT.Task,
       jobMeta.workerPaths,
       Some(versionSupport.version),
       Vector.empty,
-      jobMeta.fileResolver,
-      Logger.Quiet
```

Comment:
This change will cause Eval (in wdlTools) to use the default logger instance (set to Normal).